nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -1629,6 +1668,42 @@ private Set<String> normalizeKeyColumnNamesAndCheckForValues(RecordSchema record
         return normalizedKeyColumnNames;
     }
 
+    /*
+     * Extract list of queries from config property
+     */
+    private List<String> getQueries(final String value) {
+        if (value == null || value.isEmpty() || value.isBlank()) {
+            return null;
+        }
+        final List<String> queries = new ArrayList<>();
+        for (String query : value.split("(?<!\\\\);")) {
+            query = query.replaceAll("\\\\;", ";");
+            if (!query.isBlank()) {
+                queries.add(query.trim());
+            }
+        }
+        return queries;
+    }
+
+    /*
+     * Executes given queries using pre-defined connection.
+     * Returns null on success, or a query string if failed.
+     */
+    private Pair<String, SQLException> executeConfigStatements(final Connection con, final List<String> configQueries) {
```

Comment:
Instead of a `Pair`, created a a record with a meaningful name and meaningful variable names.